std/examples/cat.ts

```diff
@@ -5,3 +5,4 @@ for (const filename of filenames) {
   await Deno.copy(Deno.stdout, file);
   file.close();
 }
+export {};
```

Comment:
This seems very odd for the cat example. Why are these getting added everywhere?

Reply:
Sorry I didn't see your comment above.

> The TypeScript error recommends adding export {}; to the file, which is the current idiomatic way of identifying a module as a module that otherwise has no imports or exports.

I want to find a way to work around this. `export {}` doesn't make sense in these examples. I will look into it.

Reply:
I honestly don't think you can unless you really do something ugly behind the scenes.  See: https://github.com/microsoft/TypeScript/issues/18232.  There is a Stage 1 proposal to try to fix it: https://github.com/tc39/proposal-modules-pragma.

Maybe we should just ignore the error again, because we never have non-modules.

Reply:
If we ignore it, we need to check the emit on bundles... as TypeScript might not emit an async function for the module, which it didn't do in TS 3.7.